### PR TITLE
Support .. operator when binding to record fields

### PR DESCRIPTION
**Pull request by tomwhatmore**
Addresses #474

### CODE REVIEW

File: src/format.rs [hunk 1]
```diff
@@ -867,6 +894,25 @@ where
...
+    if let None = args.peek() {
+        return "()".to_doc();
+    }
+
+    break_("(", "(")
+        .append(concat(args.intersperse(delim(","))))
+        .append(break_(",", ", "))
+        .append("..")
+        .nest(INDENT)
+        .append(break_("", ""))
```
Comment (lpil) [inline src/format.rs:911]: Can we have a trailing semicolon?

```rust
break_(",", "")
```

Comment (tomwhatmore) [reply src/format.rs:911]: Do you mean comma?

Comment (lpil) [reply src/format.rs]: Sorry yes, a comma :)

Comment (tomwhatmore) [reply src/format.rs]: Done!

**Comment by lpil** [comment]
I've rebased these commits onto master as I want to avoid merge commits. Thanks!

**Pull request closed without merging**